internal/bminventory/inventory.go

```diff
@@ -937,7 +937,12 @@ func (b *bareMetalInventory) updateExternalImageInfo(ctx context.Context, infraE
 		}
 
 		details := b.getIgnitionConfigForLogging(ctx, infraEnv, b.log, imageType)
-		eventgen.SendImageInfoUpdatedEvent(ctx, b.eventsHandler, &infraEnv.ClusterID, *infraEnv.ID, details)
```

Comment:
we also have the same problem with SendHostDeregisteredEvent() in the code so it worth fixing it in the same PR.
Since it is used in 2 places I would make an auxiliary function ClusterIDPtr that returns the pointer (much likes swag.String but returns nil)

Reply:
Done.